dependencies.yaml

```diff
@@ -176,10 +182,20 @@ dependencies:
               - *libcusolver114
               - *libcusparse_dev114
               - *libcusparse114
-  doc:
+  docs:
     common:
       - output_types: [conda, requirements]
         packages:
+          - graphviz
+          - ipython
+          - ipykernel
+          - nbsphinx
+          - numpydoc
+          - pydata-sphinx-theme
+          - recommonmark
+          - scikit-learn=0.24
```

Comment:
```suggestion
          - scikit-learn=1.2
```

Reply:
Thanks! I just merged the latest changes into this PR which looks like it will incorporate this `scikit-learn` update.